src/objects/index.ts

```diff
@@ -1,15 +1,21 @@
-export type User = Agent | Customer;
+import { 
+  Customer as CustomerCustomer,
+  Agent as CustomerAgent,
+  Statistics as CustomerStatistics,
+  Geolocation as CustomerGeolocation,
+  LastPage as CustomerLastPage,
+  Visit as CustomerVisit,
+ } from "../agent/structures";
+ import {
+  Customer as AgentCustomer,
+  Agent as AgentAgent,
+ } from "../customer/structures";
```

Comment:
Shouldn't these names be reversed? I mean, import `CustomerX` from `../customer/structures` and not from `../agent/structures`

Reply:
That will do